test/normalization/text/bare-footnote.js

```diff
@@ -0,0 +1,11 @@
+/** @jsx h */
+
+export const checkSelection = false
+
+export const input = <editor>
+    <footnote>Footnote</footnote>
+</editor>
+
+export const output = <editor>
+    <p><text/></p>
```

Comment:
This is one of Slate's core normalizations (the editor may contain only block nodes), and it is also how this was handled before upgrade (see https://github.com/openstax-poland/cnx-designer/blob/master/test/plugins/footnote/schema-bare-footnote.js).

As for whether this is the intended behaviour or not, I believe this test was originally added to ensure that no content will be lost after loading an incorrect CNXML, but since this is now being handled before the document is passed to Slate, and I don't see any way this could occur during normal use, I'm not even sure if we care what happens in this situation.

Reply:
If deserialization makes sure that content will not be lost in such a case and If this can't happen by user interaction then I think we don't have to care about this.
I'm not sure if we have to have a test to cover Slate's core normalizations in that case

Reply:
I think we should not have tests for slate's core normalizations or if we want them then leave a comment that they are from the slate's core

Reply:
This test is already removed.